receiver/mongodbreceiver/testdata/integration/Dockerfile.mongodb.4_4lpu

```diff
@@ -1,4 +1,4 @@
-FROM mongo:4.4
+FROM mongo:7.0
```

Comment:
this image should not be updated since it's used to test this version of mongodb.